scripts/itest_parallel.sh

```diff
@@ -6,13 +6,19 @@ TRANCHES=$2
 TEST_FLAGS=$3
 ITEST_FLAGS=$4
 
+# Here we also shift 4 times and get the rest of our flags to pass on in $@.
+shift
+shift
+shift
+shift
+
 # Create a variable to hold the final exit code.
 exit_code=0
 
 # Run commands using xargs in parallel and capture their PIDs
 pids=()
 for ((i=0; i<PROCESSES; i++)); do 
-    scripts/itest_part.sh $i $TRANCHES $TEST_FLAGS $ITEST_FLAGS &
+    scripts/itest_part.sh $i $TRANCHES $TEST_FLAGS $ITEST_FLAGS $@ &
```

Comment:
I think we should instead put `$(ITEST_FLAGS)` and `$(TEST_FLAGS)` in quotes in the makefile, as that's the way this was probably intended to work.

Reply:
Or merge `TEST_FLAGS=$3` and `ITEST_FLAGS=$4` into just one. Because `$(TEST_FLAGS)` in the makefile might also contain multiple entries.

Reply:
I merged them the same way [itest_part.sh](https://github.com/lightningnetwork/lnd/blob/cc9e2b783ec983fbe3995d47d31898aa122b11c8/scripts/itest_part.sh#L6-L21) does it. Does this look better?